File: principal.c
#include "sumres.h"
#include "muldiv.h"

int main(void) {

    int a = 6;
    int b = 4;

    printf(" suma de %d y %d: %d\n", a, b, suma(a, b));
    printf("resta de %d y %d: %d\n", a, b, resta(a, b));
    printf("resta de %d y %d: %d\n", b, a, resta(b, a));

    a = 34;
    b = 4;

    printf("MUL de %d y %d: %d\n", a, b, multiplica(a, b));
    printf("MUL de %d y %d: %d\n", b, a, multiplica(b, a));

    a = 3;
    b = 5;
    printf("MUL de %d y %d: %d\n", a, b, multiplica(a, b));
    printf("MUL de %d y %d: %d\n", b, a, multiplica(b, a));

    printf("DIV de %d y %d: %d\n", a, b, divide(a, b));
    printf("DIV de %d y %d: %d\n", b, a, divide(b, a));

    a = 4;
    b = 34;
    printf("DIV de %d y %d: %d\n", a, b, divide(a, b));
    printf("DIV de %d y %d: %d\n", b, a, divide(b, a));

    a = 4;
    b = 4;
    printf("DIV de %d y %d: %d\n", a, b, divide(a, b));
    printf("DIV de %d y %d: %d\n", b, a, divide(b, a));
}

File: w.python/principal.py
import sumres
import muldiv

if __name__ == "__main__":

    a = 6
    b = 4

    print(" suma de", a, "y", b, ":", sumres.suma(a, b))
    print("resta de", a, "y", b, ":", sumres.resta(a, b))
    print("resta de", b, "y", a, ":", sumres.resta(b, a))

    print("  mul de", a, "y", b, ":", muldiv.multiplica(a, b))
    print("   // de", a, "y", b, ":", muldiv.divide(a, b))
    print("   // de", b, "y", a, ":", muldiv.divide(b, a))
